utils/validators/cluster-name.js

```diff
@@ -0,0 +1,10 @@
+export function clusterName(pathValue, getters, errors, validatorArgs, displayKey){
+const isRke2 = validatorArgs[0] === 'true'
+    if(isRke2){
+        if(pathValue.match(/^c-.{5}|local/gi)){
```

Comment:
Only exactly local or exactly c-xxxxx.  "c-foobar" and "locally" are allowed.  
 
https://github.com/rancher/rancher/pull/33695/files#diff-1de92fae2ed9cdb8c91c0bb0692270720b071a7426fc0f0a4ed731b9655f75b9R40